### PR TITLE
R spec testing setup

### PR DESCRIPTION
**Pull request by samu80**

### CODE REVIEW

File: app/models/problem.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Problem < ApplicationRecord
+  # belongs_to :user
```
Comment (vaibhav8186) [inline app/models/problem.rb:2]: @samu80   Remove commented code.

File: app/controllers/languages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class LanguagesController < ApplicationController
+  def index
+    response = JudgeZeroApi.new.get('/languages')
+    render json: response
+  end
+  
+  def all
```
Comment (vaibhav8186) [inline app/controllers/languages_controller.rb:7]: @samu80   If we have the `index` method then what is the use of `all`  method here.

Comment (samu80) [reply app/controllers/languages_controller.rb:7]: We are using that method for Judge0 API

**Comment by rahulknojha** [comment]
closing because it is stale now

**Pull request closed without merging**